projects/packages/forms/src/blocks/contact-form/components/jetpack-integration-controls.js

```diff
@@ -17,6 +18,12 @@ import { useIntegrationsStatus } from './jetpack-integrations-modal/hooks/useInt
 export default function IntegrationControls( { attributes, setAttributes } ) {
 	const [ isModalOpen, setIsModalOpen ] = useState( false );
 	const { integrations, refreshIntegrations } = useIntegrationsStatus();
+	const { tracks } = useAnalytics();
+
+	const handleOpenModal = () => {
+		tracks.recordEvent( 'jetpack_forms_block_modal_view' );
```

Comment:
It may be useful to have a parameter about where the click happened, so we can see which entry point is preferred in general.

Reply:
Yep, I originally had this locally but decided it may be more detail than we needed. I'm happy to add it back though.